pkg/controller/pxcbackup/controller.go

```diff
@@ -198,17 +198,28 @@ func (r *ReconcilePerconaXtraDBClusterBackup) Reconcile(ctx context.Context, req
 	}()
 
 	if err := r.checkDeadlines(ctx, cluster, cr); err != nil {
+		// There was an error with checking the deadline itself, we must retry.
```

Comment:
is it possible to introduce a scenario in the controller test to cover this?

Reply:
also the job deletion I think can be tested as well

Reply:
By controller test you mean e2e test, correct?

Reply:
Or do you mean the `controller_test.go` integration test like we have for pxc controller? This is missing in the pxcbackup controller, so I can add it

Reply:
yeap I meant `controller_test.go`, was just too bored to write the file name properly :)